src/constants/api.ts

```diff
@@ -3,3 +3,5 @@ export const API = `${HOST}/api/v1`;
 export const API_GET_PRODUCTS = `${API}/products`;
 export const API_GET_PRODUCT = (id: string): string =>
   `${API_GET_PRODUCTS}/${id}`;
+export const API_SEARCH_PRODUCT = (searchTerm: string) =>
+  `${API_GET_PRODUCTS}?search=${searchTerm}`;
```

Comment:
maybe one thing we need to think about - and it's something that could be asked to the rest of the team is if we want to have a limit (like limiting results only to a specific number, like 5 or 10)

Reply:
you could ask the rest of the group what they think so it can be decided as a team